custom-completions/windows/windows-completions.nu

```diff
@@ -0,0 +1,61 @@
+export module win {
+    # do I have admin rights?
+    export def "am i admin" [] {
```

Comment:
nushell already has a `is-admin` command that should serve this purpose

Reply:
removed